### DISCUSSION TITLE
The new version does not have TOBITMAP

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#657] [Help]
Why is there no tobitmap function in the latest 7.19? How can I reproduce this? I need to directly process the image and output it to the PICBOX. This function is very troublesome. What if I want to do TOBITMAP?

**Comment by dlemstra** [comment]
If you had used the search function you would have found this discussion: https://github.com/dlemstra/Magick.NET/discussions/652. You will need to install the `Magick.NET.SystemDrawing` library and then you will get an extension method called `ToBitMap()` in the `MagickImage` class.

**Reply by jueqing1015** [reply]
Magick.NET.SystemDrawing has been installed, the specific operation is still not clear, can you give an example?

**Reply by dlemstra** [reply]
You can do `image.ToBitMap()` after installing that package.

**Reply by jueqing1015** [reply]
![e](https://user-images.githubusercontent.com/65677359/84157414-4b694900-aa9d-11ea-91ef-f1e605f42397.jpg)
Is this the case? Sorry, my programming directory is not too proficient. I have cited the file, but there is still no TOBITMAP

**Reply by dlemstra** [reply]
Maybe you are missing a `using Imagemagick` in your code file? Or you are using the `IMagickImage` interface instead of `IMagickImage<byte>`?

**Reply by jueqing1015** [reply]
Already quoted, but I don’t know why there is no TOBITMAP
using ImageMagick;
MagickImage imag = new MagickImage("0001.tif");

**Reply by dlemstra** [reply]
You should be able to do `imag.ToBitmap()` to create the bitmap. There is probably something wrong in your project that prevents you from doing that?

**Reply by jueqing1015** [reply]
It should not be. Because I am a newly created WINFORM to do the test, can you provide a DEMO here? Let me check it myself

**Reply by dlemstra** [reply]
Can you create a simple project and put it on GitHub to demonstrate the issue?

**Reply by jueqing1015** [reply]
I'm sorry I don't know how to create a project, I sent the source code of my project
Magick.NET-Q8-AnyCPU 7.19.0.0
Magick.NET.SystemDrawing  2.0.0.0


using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Drawing.Imaging;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using ImageMagick;
namespace image
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            MagickImage image = new MagickImage("001.tif");
           
           
        }
    }
}

**Reply by jueqing1015** [reply]
Hello! I seem to have found the problem. To implement TOBITMAP, not only need to install Magick.NET.SystemDrawing but also Magick.NET.Core

**Reply by dlemstra** [reply]
Happy you resolved your issue. Strange that you need to install `Magick.NET.Core`. It is a dependency of both `Magick.NET-Q8-AnyCPU` and `Magick.NET.Core` so it should be installed automatically.

**Reply by jueqing1015** [reply]
It is recommended that you put the solution to the problem in the update instructions so that developers can quickly understand
